src/screens/AuthedAppTileScreen.tsx

```diff
@@ -20,7 +20,7 @@ export const AuthedAppTileScreen = ({
   const webViewRef = useRef<WebView>(null);
   const { data: appConfig } = useAppConfig();
   const titleOverride =
-    appConfig?.homeTab?.appTileSettings?.appTiles[appTile.id].title;
+    appConfig?.homeTab?.appTileSettings?.appTiles[appTile.id]?.title;
```

Comment:
I'm surprised the TypeScript compiler didn't catch this for you. Can the typing be improved?